src/main/java/org/trigon/utilities/QuickSortHandler.java

```diff
@@ -0,0 +1,77 @@
+package org.trigon.utilities;
+
+import java.util.Arrays;
+import java.util.function.ToDoubleFunction;
+
+/**
+ * A class that handles sorting of an array using <a href="https://en.wikipedia.org/wiki/Quicksort">Quick Sort</a>.
+ */
```

Comment:
Explain about what it does with object and stuff

Reply:
It explains more in the constructor

Reply:
This class has no constructor...
And a rule of thumb - the constructor should explain how to create the class, and the class jdoc should explain what the class does.

Reply:
Not constructor sorry, the main `sort()` method

Reply:
The class should also explain itself, this isn't contrary